airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.test.tsx

```diff
@@ -38,7 +38,8 @@ const mockConnection: WebBackendConnectionRead = {
   sourceId: "test-source",
   destinationId: "test-destination",
   status: ConnectionStatus.active,
-  schedule: undefined,
```

Comment:
@harshithmullapudi something I noticed while testing your PR branch was that when I created a new connection with a cron schedule, the connection was successfully created but the ConnectionsHandler in the backend threw this exception when trying to track the creation event in Segment: 
```
airbyte-server-8648dfb5-jbqm2 server 2022-09-13 21:54:51 ERROR i.a.s.h.ConnectionsHandler(trackNewConnection):201 - failed while reporting usage.
airbyte-server-8648dfb5-jbqm2 server java.lang.NullPointerException: Cannot invoke "io.airbyte.config.Schedule.getTimeUnit()" because "schedule" is null
airbyte-server-8648dfb5-jbqm2 server 	at io.airbyte.config.helpers.ScheduleHelpers.getIntervalInSecond(ScheduleHelpers.java:51) ~[config-models-0.40.5.jar:?]
airbyte-server-8648dfb5-jbqm2 server 	at io.airbyte.server.handlers.ConnectionsHandler.generateMetadata(ConnectionsHandler.java:223) ~[airbyte-server-0.40.5.jar:?]
airbyte-server-8648dfb5-jbqm2 server 	at io.airbyte.server.handlers.ConnectionsHandler.trackNewConnection(ConnectionsHandler.java:198) ~[airbyte-server-0.40.5.jar:?]
airbyte-server-8648dfb5-jbqm2 server 	at io.airbyte.server.handlers.ConnectionsHandler.createConnection(ConnectionsHandler.java:159) ~[airbyte-server-0.40.5.jar:?]
airbyte-server-8648dfb5-jbqm2 server 	at io.airbyte.server.handlers.WebBackendConnectionsHandler.webBackendCreateConnection(WebBackendConnectionsHandler.java:358) ~[airbyte-server-0.40.5.jar:?]
```
This seems to be caused by this line, since it is grabbing the `schedule` field of the connection which is no longer being set: https://github.com/airbytehq/airbyte/blob/39b816eb024e4583770672810f3078acb2318b73/airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java#L223
So it looks like that will need to get fixed to work with the new schedule fields before we merge in this PR, or else out tracking of new connection creations will not work properly.

Also tagging @mfsiega-airbyte since I believe you worked on some of the backend changes around these schedule fields

Reply:
I *think* this should be fixed in https://github.com/airbytehq/airbyte/pull/16590 - did your test include that PR?

Do let me know if it's still happening, since I'm operating under the belief that this is fixed :)

Reply:
I was testing this on `frontend-dev`. I just tried it again and that error is still happening, and I double-checked that frontend dev is deployed with version 0.40.5, which looks like it includes your change.

@mfsiega-airbyte If you look at the line of code I linked above, it seems to still be there in master despite your change, and that line is failing because `standardSync.getSchedule()` returns `null`, which causes `getIntervalInSecond()` to throw an NPE.

So I think that still needs to be fixed

Reply:
@mfsiega-airbyte Approved!